sympy/vector/tests/test_implicitregion.py

```diff
@@ -52,12 +56,12 @@ def test_rational_parametrization():
     assert line.rational_parametrization() == (x, 3*x + 2)
 
     circle1 = ImplicitRegion((x, y), ((x-2)**2 + (y+3)**2 - 4))
-    assert circle1.rational_parametrization(parameters=t) == (4/(t**2 + 1), 4*t/(t**2 + 1) - 3)
+    assert circle1.rational_parametrization(parameters=t) == (4 - 4/(t**2 + 1), -4*t/(t**2 + 1) - 3)
```

Comment:
I have made changes to the test cases as the result of the `rational_parametrization` method depends upon the regular point returned by `reg_point`.

Reply:
Not a problem as long as it's deterministic.